packages/bot-web-ui/src/pages/dashboard/dashboard.scss

```diff
@@ -305,10 +181,8 @@
         }
 
         &__centered {
-            display: flex;
+            @include flex-center;
             flex-direction: column;
```

Comment:
as per BEM `__centered` is not an element, it is a modifier and should be `--centered`

Reply:
@maryia-matskevich-deriv same here... `__centered` is an element, but you can see `--listed` and `--not-listed` are being applied dynamically which are sort of changing the behavior so these are the perfect fits for the modifier.

![image](https://github.com/binary-com/deriv-app/assets/90243468/ba0d7177-d9c2-4482-a4d6-0206839a52c0)

Reply:
@sandeep-deriv in this case element name does not match BEM, and must be a noun